### PR TITLE
Insurance wad maths

### PR DESCRIPTION
**Pull request by raymogg**
# Motivation
`Insurance.sol` needs to support WAD maths.

# Changes
- move all multiplication to WAD maths
- make funding rate be a WAD
- remove `INSURANCE_MUL_FACTOR` param

### CODE REVIEW

**Review by CalabashSquash — APPROVED** [review]
:myanmar:

**Review by CalabashSquash — APPROVED** [review]

**Review by sporejack — COMMENTED** [review]
LGTM!

**Review by kumar-ish — APPROVED** [review]
:rocket:

File: contracts/TracerPerpetualSwaps.sol [hunk 1]
```diff
@@ -405,9 +405,8 @@ contract TracerPerpetualSwaps is
                 // calc and pay insurance funding rate
                 // todo CASTING CHECK
                 int256 changeInInsuranceBalance =
-                    ((currentInsuranceGlobalRate - currentInsuranceUserRate) *
-                        accountBalance.totalLeveragedValue.toInt256()) /
-                        insuranceContract.INSURANCE_MUL_FACTOR();
+                    (currentInsuranceGlobalRate - currentInsuranceUserRate) *
+                        accountBalance.totalLeveragedValue.toInt256();
```
Comment (kumar-ish) [inline contracts/TracerPerpetualSwaps.sol:409]: Just a note: need to do WAD maths here

**Pull request merged**